episode-6.srt

```diff
@@ -46,7 +46,7 @@
 10
 00:00:44,100 --> 00:00:45,680
 [ru] В предыдущих сериях:
-[en] Previously on…
+[en] Previously...
```

Comment:
Thanks! It was a problem with original translation was not really correct. Fixed now on 3236c54a7e15b2b1a96bcb9bb8c7ad61e624db56.